src/documents/tutorial-userInterfaceOptions/IntegratingUIOptionsStylingPreferences.md

```diff
@@ -0,0 +1,96 @@
+related enactor will provide its own stylesheet(s), so that an integrator can import only the styles for the enactors
+they are using in their particular configuration. Althernatively the `Enactors` or `Enactors_base` stylesheets can be
+used to quickly access all of the available styling enactors styles.
+
+## Enactor Stylesheets
+
+### Base Styleheets
+
+The styling related enactors use [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)
+as hooks for applying the styling. Some enactors such as `Text Size`, set their CSS custom properties via JavaScript,
+others supply them via classes in their related `*_base` stylesheet. For example the `Enhance Inputs` preference uses
+the `EnhanceInputs_base` stylesheet. An integrator can hook into the enactor's CSS custom properties to implement their
+own styles, allowing the site to automatically adapt along with the preference changes.
+
+### Drop-in Style Application
+
+To allow for quicker initial integration with UI Options, an integrator may choose to use the drop-in stylesheets, which
+will additionally provide a set of pre-designed styles to apply the prefernce. For example the `Enhance Inputs`
+preference provides the `EnhanceInputs.css` stylesheet. `Enactors.css` will provide the complete set of enactors styles
+with their application; similar to how previous versions of Infusion supported styling for enactors.
+
+<div class="infusion-docs-note">
+
+<strong>Note:</strong> The drop-in styles are necessarily heavy handed, and include <code>!important</code> to attempt
+to override default styles. If the drop-in styles need to be overridden, it may be better to use the <code>*_base</code>
+stylesheets and build up custom styles for the site.
+
+</div>
+
+## CSS Custom Properties
+
+### Contrast
+
+The contrast CSS custom properties are set when one of the contrast classes (e.g. `.fl-theme-bw`) is applied.
+
+* `--fl-fgColor`: foreground colour, often text colour
+* `--fl-bgColor`: background colour
+* `--fl-linkColor`: text colour for links
+* `--fl-disabledColor`: text colour for disabled inputs
+* `--fl-selectedFgColor`: colour for selected text
+* `--fl-selectedBgColor`: selection background colour
+* `--fl-buttonFgColor`: text colour for buttons
+* `--fl-buttonBgColor`: background colour for buttons
+
+### Enhance Inputs
+
+The CSS Custom properties added by the `.fl-input-enhanced` class:
+
+* `--fl-enhance-font-size-factor`: (default is 1.25) the factor of the font-size increase, can be used with `calc`.
+* `--fl-enhance-font-size`: (default is 125%) the font-size value
+* `--fl-enhance-font-weight`: (default is bold) font-weight value
+* `--fl-enhance-text-decoration`: (default is underline) the text-decoration value. Used to enhance links.
+
+### Font
+
+Makes use of the `utils/_fonts.scss` for mixin functions; which may be useful if defining custom fonts for use with the
```

Comment:
Given it's the first mention in this document, may be helpful to direct the reader to some page we have that explains our Sass build process and/or explanation of what a mixin function is. Are they actually functions?